### PR TITLE
[Emu3] Add Emu3 model inference and SFT code 

### PR DESCRIPTION
**Pull request by chenyingshu**
# What does this PR do?

Adds # (feature) Emu3 model in `examples/emu3`
- inference script for (1) image reconstruction, (2) text-to-image inference, (3) VQA inference
- sft code and scripts for (1) text-to-image and (2) vqa/qa tasks.
- Differences from original [Emu3](https://github.com/baaivision/Emu3):
    - apply sequence parallel training to mindspore version.
    - mindspore ver. supports vqa/text-qa SFT, Emu3 repo does not provide code for vqa sft.

Adds # (feature) in `mindone/transformers` add:
- new logit processor: UnbatchedClassifierFreeGuidanceLogitsProcessor in `mindone/transformers/generation/logits_process.py` 
- new scheduler: get_cosine_with_min_lr_schedule_with_warmup in `mindone/transformers/optimization.py`

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/mindspore-lab/mindone/blob/master/CONTRIBUTING.md)?
- [x] Did you make sure to update the documentation with your changes? E.g. record bug fixes or new features in `What's New`. Here are the
      [documentation guidelines](https://github.com/mindspore-lab/mindcv/wiki/%E6%96%87%E6%A1%A3%E7%BC%96%E5%86%99%E8%A7%84%E8%8C%83)
- [x] Did you build and run the code without any errors?
- [x] Did you report the running environment (NPU type/MS version) and performance in the doc? (better record it for data loading, model inference, or training tasks)
- [x] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

@SamitHuang @CaitinZhao

### CODE REVIEW

**Review by SamitHuang — COMMENTED** [review]
pls rebase to the master branch to rm changes on janus and stepvideo
